integrations/anthropic/src/actions/generate-content.ts

```diff
@@ -1,7 +1,8 @@
 import Anthropic from '@anthropic-ai/sdk'
 import { MessageCreateParams, MessageCreateParamsNonStreaming } from '@anthropic-ai/sdk/resources/messages'
-import { InvalidPayloadError, UpstreamProviderError } from '@botpress/client'
+import { InvalidPayloadError } from '@botpress/client'
 import { llm } from '@botpress/common'
+import { upstreamProviderFailedError } from '@botpress/common/src/llm/errors'
```

Comment:
**chore:** no need to import from `@botpress/common/src`, just re-export what you need in common

Reply:
Good catch, addressed here: https://github.com/botpress/botpress/pull/13537/commits/5c5f4b4043edd814ab6b1b86b29a20c7e116f9e0